inlang/source-code/paraglide/paraglide-js/src/compilerV2/aliases.ts

```diff
@@ -0,0 +1,38 @@
+import type { BundleNested } from "@inlang/sdk2"
+
+/**
+ * Returns re-export statements for each alias of a MessageBundle.
+ * If no aliases are present, this function returns an empty string.
+ *
+ * @param bundle
+ */
+export function reexportAliases(bundle: BundleNested) {
+	const aliases = Object.values(bundle.alias)
+	if (aliases.length > 1) throw new Error("Only one alias is allowed per bundle") // really?
```

Comment:
No. You can compile all aliases that are valid js identifiers.

Reply:
The logic here should change. Compile every alias that is a valid JS identifier. The concept of a default alias has been removed. See https://github.com/opral/monorepo/pull/3048#discussion_r1707395555 for context